### PR TITLE
Issue1742 hide speaker list

### PR DESCRIPTION
**Pull request by r-peschke**
separate LoF.can_be_speaker from LoF.can_see, migration to add LoF.can_see explicitly
Projector.is_internal field in models and actions

### CODE REVIEW

**Review by jsangmeister — CHANGES_REQUESTED** [review]

**Comment by jsangmeister** [comment]
Also, please stick to https://github.com/OpenSlides/OpenSlides/wiki/Development-organization#pull-request-titles-and-commit-messages

File: openslides_backend/migrations/migrations/0041_add_ListOfSpeakers_CAN_SEE_to_group_events.py
[the remark this answers is not in the document]
Comment (r-peschke) [reply openslides_backend/migrations/migrations/0041_add_ListOfSpeakers_CAN_SEE_to_group_events.py:1]: renamed

**Comment by ostcar** [comment]
The models.yml has changed in the main branch.

Could you merge the main branch in the PR so I can use the models.yml from this branch to make the necessary changes in the autoupdate service?

**Comment by luisa-beerboom** [comment]
I should not be able to set the meetings reference projector to internal or set an internal projector to be the new reference projector. It will break the autopilot.

I can do that right now.

File: openslides_backend/migrations/migrations/0041_add_ListOfSpeakers_CAN_SEE_to_group_events.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                "assignment_poll_default_100_percent_base",
+                "assignment_poll_default_onehundred_percent_base",
+            ),
+            ("poll_default_100_percent_base", "poll_default_onehundred_percent_base"),
+        ],
+    }
+
+    def modify(self, object: Dict[str, Any]) -> None:
+        if "list_of_speakers.can_be_speaker" in object["permissions"]:
+            object["permissions"].append("list_of_speakers.can_see")
```
[the remark this answers is not in the document]
Comment (r-peschke) [reply openslides_backend/migrations/migrations/0041_add_ListOfSpeakers_CAN_SEE_to_group_events.py]: done

**Review by jsangmeister — APPROVED** [review]

**Pull request merged**